layouter/templates/layouter/container.html

```diff
@@ -2,14 +2,38 @@
 {% load staticfiles %}
 <div class="row layouterRow--outer">
     <div class="col-md-offset-{{ instance.margin }} col-md-{{ width }}">
-        <div class="row layouterRow--inner {% if request.session.layouter_grid %}show-grid{% endif %} {{ instance.css_classes }}">
+        <div class="row layouterRow--inner
+                    {% if request.session.layouter_grid %}show-grid{% endif %}
+                    {% if instance.background_image_parallax == instance.CSS_PARALLAX %}layouterRow--parallax{% endif %}
+                    {% if instance.background_image_parallax == instance.TRUE_PARALLAX %}parallax-window{% endif %}
+                    {{ instance.css_classes }}"
+            {% if instance.background_image %}
+                {% if instance.background_image_parallax == instance.TRUE_PARALLAX %}
+                    {# Setup True / JS Parallax via parallax.js #}
+                    data-parallax="scroll" data-image-src="{{ instance.get_resized_background_image.url }}"
+                    data-naturalWidth="{{ instance.get_resized_background_image.width }}"
+                    data-naturalHeigth="{{ instance.get_resized_background_image.height }}"
+                {% else %}
+                    {# CSS_PARALLAX also uses the background image #}
+                    style="background-image: url('{{ instance.get_resized_background_image.url }}')"
+                {% endif %}
+            {% endif %}
+        >
             {% include 'layouter/partials/one_col.html' %}
             {% include 'layouter/partials/two_cols.html' %}
             {% include 'layouter/partials/three_cols.html' %}
             {% include 'layouter/partials/four_cols.html' %}
         </div>
     </div>
 </div>
+
+{% if instance.background_image_parallax == instance.TRUE_PARALLAX %}
```

Comment:
This is nice. When I read about the feature, this was actually the first thing that came to my mind - only use the .js if needed. (y)